### PR TITLE
cargo: bump version-sync dependency

### PR DESCRIPTION
**Pull request by mgeisler**

### CODE REVIEW

**Comment by stbuehler** [comment]
Hi.

Is there a reason you're pushing for an update?

I'm not sure whether bumping a dev-dependency also requires bumping on my side (i.e. what the "costs" are), and don't see the benefits.

**Comment by mgeisler** [comment]
Hi @stbuehler, that's a very good question :-) Since version-sync is just a tiny helper crate, you don't need to worry about bumping the version you use.

However, my thinking from my own projects is that I might as well use the "latest and greatest" version _as long as they don't introduce new requirements_. Your project already requires Rust 2018, so you already depend on a new enough compiler to use version-sync 0.8.0. This should mean that the version bump is completely transparent to your users.

Looking at the bigger picture, it's actually a matter of me not having released a version 1.0.0 yet. If I had, then you would automatically get new 1.x.y releases, assuming you specify the dependency as `version-sync = "1.0"` or similar.

Long story short: feel free to stick with the current version :-)

**Comment by stbuehler** [comment]
Raised to 0.9.1 in 6ae119303a74088ecb4bb3dbd90e4f3e3bb72b0f on the master branch, which should end up in a 0.2.* release.
